research-software-directory/RSD-as-a-service
# Organisation table

# Organisation table

Changes proposed in this pull request:

* add organisation table
* add status for inter relationships: can either be requested by one side or approved by both sides
* some tests are added
* not related but needed: change flag `is_corporate_blog` to `is_featured` for mentions

How to test:

* build and run the tests, there should be no failures and 67 assertions
* you can make requests to /organisation, /software_for_organisation and /project_for_organisation
* data migration should still work

PR Checklist:


closes #35
